samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -722,6 +755,11 @@ CHIP_ERROR BLEConnectivityManager::RemoveBLEProvider(bt_addr_le_t address)
 		return CHIP_ERROR_INTERNAL;
 	}
 
+	if (0 == mConnectedProvidersCounter) {
+		/* There are no active providers anymore, so disable the Connected state */
+		UpdateStateFlag(State::Connected, false);
```

Comment:
Maybe it's a minor comment, but according to description:

> Connected - All connections to bridged devices are stable.

shouldn't we then set `Connected` state to false on the first `Disconnect` event, not in the situation if all providers were removed?

Reply:
Why? Some of the existing providers can be still stable. If the Bridge loses connection, then the DeviceLost flag will be set, and since it has a bigger priority than the `Connected` state it will be signalized, If we remove this device the `DeviceLost` state will be disabled. If it still has some bridged device the `Connected` state will be signalized automatically after that. Otherwise, if we clear the `Connected` on the first `Disconnect` event, the Bridge won't signalize that other bridged devices are connected when all states with higher priority are disabled. So the description is also correct, because we can signalize the `Connected` state only if all devices are stable, and there is no scanning or pairing in progress.